pyvo/registry/regtap.py

```diff
@@ -208,6 +323,55 @@ class RegistryResource(dalq.Record):
 
     _service = None
 
+    # the following attribute is used by datasearch._build_regtap_query
+    # to figure build the select clause; it is maintained here
+    # because this class knows what it expects to get.
+    #
+    # Each item is either a plain string for a column name, or
+    # a 2-tuple for an as clause; all plain strings are used
+    # used in the group by, and so it is assumed they are
+    # 1:1 to ivoid.
+    expected_columns = [
+        "ivoid",
+        "res_type",
+        "short_name",
+        "res_title",
+        "content_level",
+        "res_description",
+        "reference_url",
+        "creator_seq",
+        "content_type",
+        "source_format",
```

Comment:
Shouldn't `source_value` be here too? (or maybe in `to_table`). I am not sure about their exact meaning, but it is another thing that breaks few cells in the tutorial notebooks.